projects/core/src/configurable-routes/routes-config.ts

```diff
@@ -0,0 +1,81 @@
+export interface RoutesTranslations {
```

Comment:
We might have discussed already, but wondering how a library consumer (industry team, customer) woudl be able to extend this config and have strong types.

Reply:
He will take advantage of typeahead while typing standard page names in config in app.module. However the is an opportunity for him do use any other page names that he wants (line 20), but for them he will not have typeahead.
So this interface it's not strongly typed but helps as much as it can.